lib/source/Instruction.cpp

```diff
@@ -1570,6 +1575,20 @@ spvgentwo::Instruction* spvgentwo::Instruction::opConvertPtrToU(Instruction* _pP
 	return error();
 }
 
+spvgentwo::Instruction* spvgentwo::Instruction::opConvertUToPtr(Instruction* _pUInt,
+	const Type& _type)
+{
+	if (_type.isPointer() && _type.getStorageClass() == spv::StorageClass::PhysicalStorageBuffer)
```

Comment:
please always validate all input types, here the type of _pUInt operand needs to be checked for Integer & Unsigned